src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Application.java

```diff
@@ -496,6 +496,12 @@ public class Application {
     )
     private YesOrNo switchedToSoleCo;
 
+    @CCD(
+        label = "Is a required admin clarification now submitted?",
+        access = {CaseworkerAccess.class}
+    )
```

Comment:
Should this field be in ConditionalOrder class ?

Reply:
Relocated.